docs/method/render.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+pandoc -V geometry:margin=1in -o locuszoom-credible-sets.pdf locuszoom-credible-sets.md
```

Comment:
This description of the method is a big, and readable, improvement- thanks!

If you think this will get revised in the future, it could conceivably be added as an npm script so that all maintenance operations for the repo can be found and run from one consistent place.

Reply:
Could it be included in the npm docs command? I'm not sure everyone will have pandoc installed on their machine, I guess is the only issue, and npm probably can't pull it in (I'm assuming?)

Reply:
In theory NPM can run any command on your system, not just node modules. 

I've added a "user beware" message to the README about the missing dependencies- since it also requires a latex install, I avoided even trying to impose assumptions on the user. :)